apps/console.py

```diff
@@ -356,7 +363,7 @@ def show_services(self, services):
         self.services_text.text = lines
         self.ui.invalidate()
 
-    async def show_attributes(self, attributes):
+    def show_attributes(self, attributes):
```

Comment:
Drive by fix: this was the only show_* method that was async, but it doesn't need to be.